src/core/include/openvino/op/clamp.hpp

```diff
@@ -47,9 +47,9 @@ class OPENVINO_API Clamp : public util::UnaryElementwiseArithmetic {
     }
     OPENVINO_SUPPRESS_DEPRECATED_START
     bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Can we migrate to ov::Tensor?

Reply:
I've add evaluate for ov::Tensor only there where looks is needed mostly in cases when there is a case for inputs which can have Shape{0} or Shape{max} which can trigger exception when converting ov::Tensor to HostTensor in base evaluate method in Node class.
And also it looks like the old version of evaluate cannot be dropped because of template plugin tests.

Reply:
Why do you have a problem with template functional tests?

Reply:
I think is related to line [evaluate in template](https://github.com/openvinotoolkit/openvino/blob/3ad92084d69d0a9a143b244e139890f8df78e552/src/plugins/template/backend/int_executable.cpp#L163) and work is ongoing in #13944